### PR TITLE
update zh-cn (i18n)

### PR DESCRIPTION
**Pull request by hoilc**
```
> php cli\check.translation.php -d -l=zh-cn -r
Language: zh-cn

zh-cn - Translation is 100.0% complete.
```

### CODE REVIEW

File: app/i18n/zh-cn/install.php [hunk 1]
```diff
@@ -69,8 +69,8 @@
 			'ok' => '已找到推荐的 JSON 解析库。',
 		),
 		'mbstring' => array(
-			'nok' => 'Cannot find the recommended library mbstring for Unicode.',	//TODO - Translation
-			'ok' => 'You have the recommended library mbstring for Unicode.',	//TODO - Translation
+			'nok' => '找不到推荐的 Unicode 解析库 (mbstring)。',
```
Comment (Frenzie) [inline app/i18n/zh-cn/install.php]:
```suggestion
			'nok' => '找不到推荐的 Unicode 解析库 (mbstring)。',
```

File: app/i18n/zh-cn/conf.php [hunk 2]
```diff
@@ -28,7 +28,7 @@
 			'seconds' => '秒 (0 表示不超时)',
 			'timeout' => 'HTML5 通知超时时间',
 		),
-		'show_nav_buttons' => 'Show the navigation buttons',	//TODO - Translation
+		'show_nav_buttons' => '显示导航按钮',
```
Comment (Frenzie) [inline app/i18n/zh-cn/conf.php]:
```suggestion
		'show_nav_buttons' => '显示导航按钮',
```

File: app/i18n/zh-cn/conf.php [hunk 3]
```diff
@@ -148,31 +148,31 @@
...
-		'previous_article' => '跳转到上一篇文章',
+		'previous_article' => '打开上一篇文章',
 		'reading_view' => '切换到阅读视图',
 		'rss_view' => '在新标签中打开 RSS 视图',
 		'see_on_website' => '在原网站上查看',
 		'shift_for_all_read' => '+ <code>shift</code> 可以将全部文章设为已读',
-		'skip_next_article' => 'Focus next without opening',	//TODO - Translation
-		'skip_previous_article' => 'Focus previous without opening',	//TODO - Translation
+		'skip_next_article' => '跳转到下一篇文章而不打开',
+		'skip_previous_article' => '跳转到上一篇文章而不打开',
```
Comment (Frenzie) [inline app/i18n/zh-cn/conf.php]:
```suggestion
		'skip_previous_article' => '跳转到上一篇文章而不打开',
```

File: app/i18n/zh-cn/sub.php [hunk 4]
```diff
@@ -47,16 +47,16 @@
...
 		'websub' => 'WebSub 即时通知',
 		'show' => array(
-			'all' => 'Show all feeds',	//TODO - Translation
-			'error' => 'Show only feeds with error',	//TODO - Translation
+			'all' => '显示所有 RSS 源',
+			'error' => '仅显示有错误的 RSS 源',
 		),
 		'showing' => array(
-			'error' => 'Showing only feeds with error',	//TODO - Translation
+			'error' => '正在显示有错误的 RSS 源',
```
Comment (Frenzie) [inline app/i18n/zh-cn/sub.php]:
```suggestion
			'error' => '正在显示有错误的 RSS 源',
```

File: app/i18n/zh-cn/sub.php [hunk 5]
```diff
@@ -47,16 +47,16 @@
...
-			'error' => 'Show only feeds with error',	//TODO - Translation
+			'all' => '显示所有 RSS 源',
+			'error' => '仅显示有错误的 RSS 源',
 		),
 		'showing' => array(
-			'error' => 'Showing only feeds with error',	//TODO - Translation
+			'error' => '正在显示有错误的 RSS 源',	
 		),
-		'ssl_verify' => 'Verify SSL security',	//TODO - Translation
+		'ssl_verify' => '验证 SSL 安全',
```
Comment (Frenzie) [inline app/i18n/zh-cn/sub.php]:
```suggestion
		'ssl_verify' => '验证 SSL 安全',
```

**Review by Frenzie — COMMENTED** [review]
Thanks! Just some stray whitespace.

**Comment by aledeg** [comment]
@hoilc Thank you for your contribution. Stay tune, there is more changes to come :)

**Comment by hoilc** [comment]
> Thanks! Just some stray whitespace.

😊Fixed.

**Comment by Frenzie** [comment]
@aledeg In that case should it wait or can it be merged already? :-)

**Comment by aledeg** [comment]
We can merge that right now. I know some new translations are coming because I am adding things.
But my PR is not ready yet. There is more work to do on it.

**Pull request merged**